Flow.Launcher/Properties/PublishProfiles/Net5.0-SelfContained.pubxml

```diff
@@ -1,18 +0,0 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<!--
-https://go.microsoft.com/fwlink/?LinkID=208121. 
--->
-<Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <PropertyGroup>
-    <PublishProtocol>FileSystem</PublishProtocol>
-    <Configuration>Release</Configuration>
-    <Platform>Any CPU</Platform>
-    <TargetFramework>net5.0-windows10.0.19041.0</TargetFramework>
-    <PublishDir>..\Output\Release\</PublishDir>
-    <RuntimeIdentifier>win-x64</RuntimeIdentifier>
-    <SelfContained>true</SelfContained>
-    <PublishSingleFile>False</PublishSingleFile>
-    <PublishReadyToRun>False</PublishReadyToRun>
-    <PublishTrimmed>False</PublishTrimmed>
-  </PropertyGroup>
-</Project>
```

Comment:
hey @taooceros why is this publish profile removed? we still need it right for 6?

Reply:
I remember I rename it? I will check again.

Reply:
all good, sorted out, see commit.